vals.go

```diff
@@ -90,6 +91,7 @@ const (
 	ProviderTFStateRemote      = "tfstateremote"
 	ProviderAzureKeyVault      = "azurekeyvault"
 	ProviderEnvSubst           = "envsubst"
+	ProviderOnePassword        = "op"
```

Comment:
op? it's too sample and unreadable.

Reply:
On one hand I agree, on the other hand is not that different than `bw`. But more importantly it matches the 1Password CLI command:

```bash
op read op://development/Github/credentials/personal_token
```

See https://developer.1password.com/docs/cli/secret-references#with-op-read

This makes it a very nice experience, because whatever works in cli will work in vals.